### PR TITLE
Changed refuse policy

### PR DESCRIPTION
**Pull request by g4s8**
zerocracy/farm#1926 - changed refuse policy, penalty points depending on time of refusing

### CODE REVIEW

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this pull request is too small, just 6 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @g4s8[/z](https://www.0crat.com/u/g4s8) please review this and merge or reject

**Comment by g4s8** [comment]
@yegor256 please check this PR

File: _posts/2018/02/2018-02-22-policy.md [hunk 1]
```diff
@@ -115,7 +115,11 @@ In exceptional situations, you should ask project architect to boost your job.
 <a id="6" href="#6">§6</a>
 "Refusal."
 You may refuse to complete any job by saying `refuse` or `resign`.
-You will get a penalty of <span id="6.penalty">15</span> [points](#18) for that.
+Depends on when the job was refused, performer may get the penality:
```
Comment (Protagores) [inline _posts/2018/02/2018-02-22-policy.md]: @g4s8 There’s a minor stylistic inconsistency here. It’s supposed to be either ‘you’ or ‘performer’ in all sentences inside this section.

Also, it’s written as ‘penalty’, not ‘penality’.

**Comment by g4s8** [comment]
@Protagores thanks! fixed

**Comment by Protagores** [comment]
@g4s8 Could you please rebase into a single atomic commit?

**Comment by g4s8** [comment]
@Protagores why?

**Comment by Protagores** [comment]
@g4s8 Well, I’m not exactly aware of conventions accepted in this particular project, but as a rule of thumb it is usually preferred to keep the commit history reasonably clean, at least when it comes to such cosmetic changes.

If I’m being mistaken here, please clarify.

**Comment by g4s8** [comment]
@Protagores I think one commit per change is a good practice - you'll be able to identify the reason of commit with `git blame`, e.g. you will see that this commit was made because of code-review for #130 ticket, so you can go to this ticket and check reviews. And generally speaking, I think atomic commits are better than huge - `git blame` shows you the message of every change.

**Comment by Protagores** [comment]
@g4s8 Fair enough, but it’s sad that there’s a lack of consistency within Zerocracy managed projects. In one instance, I was being asked to rebase commits in my PR by @ilyakharlamov, see here: https://github.com/yegor256/jpeek/pull/326#pullrequestreview-231470854

@yegor256’s article about guidelines (https://www.yegor256.com/2014/04/15/github-guidelines.html) doesn’t seem to have to say anything about this, either.

**Comment by g4s8** [comment]
@Protagores but you may argue with reviewer if he/she asked to do something what is not documented. If review requested for changes you may:
 - agree and make changes
 - disagree, but make changes, if reviewer references to some project requirements, e.g. if the readme file says "make one commit per PR"
 - disagree and argue with reviewer, if requested changes was opinion based

**Comment by g4s8** [comment]
@yegor256 ping, can we merge this?

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/zerocracy.github.io#103` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request merged**